rlclientlib/schema/v2/FileFormat.fbs

```diff
@@ -0,0 +1,25 @@
+include "Event.fbs";
+
+namespace reinforcement_learning.messages.flatbuff.v2;
+
```

Comment:
I am wondering whether this should be called plain `Event` or `ProcessedEvent` instead of `JoinedEvent` since `Joined` implies that the event has found its partner(s) and they are together now in the byte array. Where as from what I can see these will just be the events plus a timestamp

Reply:
Makes sense to change the name.